cabal.project

```diff
@@ -1,5 +1,7 @@
 packages:   streamly.cabal
           , test/streamly-tests.cabal
           , benchmark/streamly-benchmarks.cabal
+--           , ../fusion-plugin
```

Comment:
This was committed by mistake I believe.

Reply:
yes. There may be things in this PR that may need some cleanup. Please cleanup where necessary.